deltachat-ios/Controller/WelcomeViewController.swift

```diff
@@ -94,8 +94,11 @@ class WelcomeViewController: UIViewController, ProgressAlertHandler {
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
         welcomeView.minContainerHeight = size.height - view.safeAreaInsets.top
-        scrollView.setContentOffset(CGPoint(x: 0, y: 0), animated: true)
-    }
+        scrollView.setContentOffset(CGPoint(x: 0, y: 0), animated: false)
+        coordinator.animate(alongsideTransition: { [weak self] _ in
+            self?.scrollView.scrollToBottom(animated: true)
+        })
+     }
```

Comment:
what is this change doing?

Reply:
ah, i see scrolling the view to bottom when switching from portrait to landscape. 

as this would be most important when the view is _started_ in landscape, but it does not happen then, i think, we can remove this extra complexity. by the  smaller image, at least one button is even visible on iphoneSE.